RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py

```diff
@@ -3,5 +3,12 @@
 pfDeepCSVJetTags = cms.EDProducer(
 	'DeepFlavourJetTagsProducer',
 	src = cms.InputTag('pfDeepCSVTagInfos'),
+  checkSVForDefaults = cms.bool(False),
+  meanPadding = cms.bool(False),
 	NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepFlavourNoSL.json')
 	)
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(pfDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
+phase1Pixel.toModify(pfDeepCSVJetTags, checkSVForDefaults = cms.bool(True))
+phase1Pixel.toModify(pfDeepCSVJetTags, meanPadding = cms.bool(True))
```

Comment:
why are these parameters changing for 2017?
Other than 4 pixel layer related logic, the same best kind of training should be provided for 2017 and the past.